pkg/apis/kubedirector.bluedata.io/v1alpha1/types.go

```diff
@@ -100,6 +101,7 @@ type MemberStatus struct {
 	Service string `json:"service"`
 	PVC     string `json:"pvc,omitempty"`
 	State   string `json:"state"`
+	NodeId  string `json:"node_id"`
 }
```

Comment:
Given the changes to the status object (and subobjects), the schema in crd-cluster.yaml will need to be updated too. (And then the [wiki](https://github.com/bluek8s/kubedirector/wiki/Type-Definitions-for-KubeDirectorCluster), which is on me.)